docs/src/main/docs/guides/91_mp-tutorial.adoc

```diff
@@ -0,0 +1,1267 @@
+                        </archive>
+                    </configuration>
+                </plugin>
+            </plugins>
+        </pluginManagement>
+
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-dependency-plugin</artifactId> <3>
+                <executions>
+                    <execution>
+                        <id>copy-dependencies</id>
+                        <phase>prepare-package</phase>
+                        <goals>
+                            <goal>copy-dependencies</goal>
+                        </goals>
+                        <configuration>
+                            <outputDirectory>${copied.libs.dir}</outputDirectory>
+                            <overWriteReleases>false</overWriteReleases>
+                            <overWriteSnapshots>false</overWriteSnapshots>
+                            <overWriteIfNewer>true</overWriteIfNewer>
+                            <overWriteIfNewer>true</overWriteIfNewer>
+                            <includeScope>runtime</includeScope>
+                            <excludeScope>test</excludeScope>
+                        </configuration>
+                    </execution>
+                </executions>
+            </plugin>
+        </plugins>
+    </build>
+
+</project>
+---- 
+
+<1> New entries are created in the `properties` to set the name of the `libs`
+ directory where the dependencies will be placed.   
+<2> In the `build` section,
+ a `plugin` entry is added to configure the `maven-jar-plugin` to set the
+ main class in the manifest, and to add the `libs` directory to the
+ manifest classpath.  
+<3> Additionally, the `maven-dependency-plugin` is
+ configured to copy the dependencies into the `libs` directory.
+
+With these updates in place, build the application with Maven:
+
+[source,bash]
+.Build the Application
+----
+mvn package
+----
+
+This will build the application jar and save all runtime 
+dependencies in the `target/libs` directory. This means you can easily start the
+ application by running the application jar file:
+
+[source,bash]
+.Run the application
+----
+java -jar target/helidon-mp-tutorial-1.0-SNAPSHOT.jar
+----
+
+At this stage, the application is a very simple "Hello World" greeting service. 
+ It supports a single GET request for generating a greeting message.
+ The response is encoded using JSON.
+ For example:
+
+[source,bash]
+.Try the Application
+----
+curl -X GET http://localhost:7001/greet
+{"message":"Hello World!"}
+----
+
+In the output you can see the JSON output from the `getDefaultMessage()`
+ method that was discussed earlier.  The server has used a default port
+ `7001`.  The application can be stopped cleanly by pressing Ctrl+C.
+
+== Configuration
+
+Helidon MP applications can use the `microprofile-config.properties` 
+ file to specify configuration data.  This file is read by default 
+ if it is present. Create this file in  
```

Comment:
Not sure how best to present this, but "if it is present" doesn't actually apply to `microprofile-config.properties`, but rather the classpath resource whose name is `META-INF/microprofile-config.properties`.